applications/tari_console_wallet/src/automation/command_parser.rs

```diff
@@ -61,6 +61,7 @@ impl Display for ParsedCommand {
             ClearCustomBaseNode => "clear-custom-base-node",
             RegisterAsset => "register-asset",
             MintTokens => "mint-tokens",
+            InitialAssetCheckpoint => "create-initial-checkpoint",
```

Comment:
Let's keep these names the same otherwise it gets confusing
```suggestion
            CreateInitialCheckpoint => "create-initial-checkpoint",
```